util/Doxyfile

```diff
@@ -0,0 +1,23 @@
+INPUT = README.md index.php boot.php testargs.php update.php mod/ object/ include/ js/ util/ view/ version.inc
+RECURSIVE = YES
+PROJECT_NAME = "Friendica"
+PROJECT_LOGO = images/friendica-64.jpg
+EXCLUDE = .htconfig.php library/ doc/ .git/ log/ util/zotsh/easywebdav/
```

Comment:
Could you have a look in the .gitignore for all folders that are excluded there to include them here as well?